DuggaSys/diagram.js

```diff
@@ -119,17 +119,21 @@ var defaults = {
     defaultERrelation: { kind: "ERRelation", fill: "White", Stroke: "Black", width: 60, height: 60 },
     defaultERattr: { kind: "ERAttr", fill: "White", Stroke: "Black", width: 90, height: 45 }
 }
+// States used for ER-elements 
+var attrState=["normal","multiple","key","computed"];
+var entityState=["normal","weak"];
+var relationState=["normal","weak"];
```

Comment:
Please use enums instead of strings to prevent accidental misspelling further down in code in the future.